Microsoft.Azure.Devices.LongHaul.Module/Parameters.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client;
+using CommandLine;
+
+namespace Microsoft.Azure.Devices.LongHaul.Module
+{
+    public enum TransportType
+    {
+        Mqtt,
+        Amqp,
+    };
+
+    public enum JsonSerializingLibrary
+    {
+        SystemTextJson,
+        NewtonsoftJson,
+    };
+
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "ConnectionString",
+            Required = false,
+            HelpText = "The connection string for the device to simulate.")]
+        public string ConnectionString { get; set; } = Environment.GetEnvironmentVariable("IOTHUB_LONG_HAUL_MODULE_CONNECTION_STRING");
```

Comment:
Don't forget to set up the env var for module connection string in [LongHaulSetup.ps1](https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/e2e/LongHaul/prerequisites/LongHaulSetup.ps1)